pkgs/development/python-modules/confusable-homoglyphs/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools,
+  unittestCheckHook,
+  wheel,
+  click,
+}:
+
+let
+  kebabToSnake = str: builtins.replaceStrings [ "-" ] [ "_" ] str;
+in
+buildPythonPackage rec {
+  pname = "confusable-homoglyphs";
+  version = "3.3.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = kebabToSnake pname;
+    hash = "sha256-uZUAHJsuG0zqDPXzhAp8eRiKjLutBT1pNXK9jBwexGA=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  disabled = pythonOlder "3.7";
+
+  optional-dependencies = {
+    cli = [ click ];
+  };
+
+  checkInputs = [ unittestCheckHook ] ++ optional-dependencies.cli;
+
+  pythonImportsCheck = [ (kebabToSnake pname) ];
```

```suggestion
  pythonImportsCheck = [ "confusable-homoglyphs" ];
```
`pname` should not be reused.
See https://github.com/NixOS/nixpkgs/issues/277994